### DISCUSSION TITLE
Auto reload pods for Argo Rollout

### DISCUSSION THREAD

**Discussion by josh-sherman-worthwhile** [Infisical/infisical#6963] [Feature Request]
### Feature description
Currently if you add the correct annotation to your Kubernetes deployment it will auto reload the pods once it detects any changes to the secret.  I'd like to have this same capability implemented for [Argo rollouts](https://argoproj.github.io/rollouts/).  I tried adding the annotation as I did with the deployment, but for the rollout it doesn't take any action currently.

### Why would it be useful?
This would prevent users from having to restart pods manually when using Argo rollouts and secrets are updated.

**Comment by vmatsiiako** [comment]
Hi @josh-sherman-worthwhile! We will try to take a look into this soon. 

cc @maidul98

**Comment by josh-sherman-worthwhile** [comment]
I just wanted to check in and see if there was any progress on this or anything I could do to assist.

**Comment by KevinYobeth** [comment]
hi is there any update regarding this? I stumble upon the same problem
